packages/graphql-hooks/index.d.ts

```diff
@@ -146,6 +159,10 @@ interface ClientOptions {
 
 type Headers = { [k: string]: string }
 
+type LocalQueries = {
+  [q: string]: (variables: object, operationName?: string) => object
```

Comment:
I think making the interface partial still wouldn't work because it would still detect that the config passed to the constructor of `LocalGraphQLClient` isn't compatible with the constructor of `GraphQLClient`

One hacky solution could be to pass an object adding a dummy `url` prop to `super` in the constructor maybe like `super({ ...config, url: '' })`

Reply:
> I think making the interface partial still wouldn't work because it would still detect that the config passed to the constructor of LocalGraphQLClient isn't compatible with the constructor of GraphQLClient

I tried locally in a typescript project and worked just fine.

Whenever we will do the migration of this codebase to Typescript, a simple casting would do the work:
`super(config as ClientOptions)`